packages/components/molecules/f-rating/src/tenants/en-GB.js

```diff
@@ -1,4 +1,7 @@
 export default {
     locale: 'en-GB',
-    text: 'I am a VRating Component (GB)'
+    ratings: {
+        starsDescription: 'stars out of',
+        starsDescriptionSingular: 'star out of'
```

Comment:
we can also do something cool here to pass the value into the message I think. Sending you an example offline

Reply:
Yeah so you can use a template. The issue from what I can see (including the related comments about plural/singular copy) is that we're not using the vue globalisation package which gives us the template helper. Also using the plural helper from i18n means we'd probably have to modify that package so it exposes it for us. Will have a look further into it tho.

Reply:
If the package we use is preventing us from properly using the i18n features then yeah we should probably look into extending the package - happy to help with this if you like!

Reply:
Looks like i18n works out the box with the `tc` version so added that :)